posix/common

```diff
@@ -12,6 +12,11 @@ if [ ! -d .git ]; then
   git config --global --add safe.directory '*'
   git remote add origin ${DRONE_REMOTE_URL}
   set +x
+else
+  set -x
+  git config --global --add safe.directory '*'
+  git remote set-url origin ${DRONE_REMOTE_URL}
+  set +x
 fi
```

Comment:
let us not set it if it is not required. It is for debugging

Reply:
Are you referring to the `set -x`? I see it is wrapping all the other commands made in bash scripts throughout this plugin's implementation. It should be useful to see which commands are running, right?